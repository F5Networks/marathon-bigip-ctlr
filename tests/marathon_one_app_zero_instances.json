[
    {
        "acceptedResourceRoles": null,
        "args": [
            "--sse",
            "--marathon",
            "http://10.141.141.10:8080",
            "--partition",
            "mesos",
            "--hostname",
            "10.128.1.145",
            "--username",
            "admin",
            "--password",
            "default"
        ],
        "backoffFactor": 1.15,
        "backoffSeconds": 1,
        "cmd": null,
        "constraints": [],
        "container": {
            "docker": {
                "forcePullImage": false,
                "image": "f5networks/marathon-bigip-ctlr:latest",
                "network": "BRIDGE",
                "parameters": [],
                "privileged": false
            },
            "type": "DOCKER",
            "volumes": []
        },
        "cpus": 0.5,
        "dependencies": [],
        "deployments": [],
        "disk": 0,
        "env": {},
        "executor": "",
        "healthChecks": [],
        "id": "/f5-service-router",
        "instances": 1,
        "labels": {},
        "maxLaunchDelaySeconds": 3600,
        "mem": 128,
        "ports": [
            10000
        ],
        "requirePorts": false,
        "storeUrls": [],
        "tasks": [
            {
                "appId": "/f5-service-router",
                "host": "10.141.141.10",
                "id": "f5-service-router.9c824d13-3890-11e6-a960-0242ed8b262a",
                "ports": [
                    31324
                ],
                "slaveId": "c0ca17c6-075f-4e12-997a-38537feb6c16-S0",
                "stagedAt": "2016-06-22T15:47:23.399Z",
                "startedAt": "2016-06-22T15:47:43.210Z",
                "version": "2016-06-22T15:47:22.754Z"
            }
        ],
        "tasksHealthy": 0,
        "tasksRunning": 1,
        "tasksStaged": 0,
        "tasksUnhealthy": 0,
        "upgradeStrategy": {
            "maximumOverCapacity": 1,
            "minimumHealthCapacity": 1
        },
        "uris": [],
        "user": null,
        "version": "2016-06-22T15:47:22.754Z",
        "versionInfo": {
            "lastConfigChangeAt": "2016-06-22T15:47:22.754Z",
            "lastScalingAt": "2016-06-22T15:47:22.754Z"
        }
    },
    {
        "acceptedResourceRoles": null,
        "args": null,
        "backoffFactor": 1.15,
        "backoffSeconds": 1,
        "cmd": null,
        "constraints": [],
        "container": {
            "docker": {
                "forcePullImage": false,
                "image": "user/node-web-app",
                "network": "BRIDGE",
                "parameters": [],
                "portMappings": [
                    {
                        "containerPort": 8088,
                        "hostPort": 0,
                        "protocol": "tcp",
                        "servicePort": 10001
                    }
                ],
                "privileged": false
            },
            "type": "DOCKER",
            "volumes": []
        },
        "cpus": 0.1,
        "dependencies": [],
        "deployments": [],
        "disk": 0,
        "env": {},
        "executor": "",
        "healthChecks": [
            {
                "gracePeriodSeconds": 5,
                "ignoreHttp1xx": false,
                "intervalSeconds": 10,
                "maxConsecutiveFailures": 1,
                "path": "/my_healthcheck",
                "portIndex": 0,
                "protocol": "HTTP",
                "timeoutSeconds": 20
            }
        ],
        "id": "/server-app",
        "instances": 0,
        "labels": {
            "F5_0_BIND_ADDR": "10.128.10.240",
            "F5_0_MODE": "http",
            "F5_0_PORT": "80",
            "F5_PARTITION": "mesos",
            "tdp-myapp": "enable"
        },
        "maxLaunchDelaySeconds": 3600,
        "mem": 16,
        "ports": [
            10001
        ],
        "requirePorts": false,
        "storeUrls": [],
        "tasks": [],
        "tasksHealthy": 0,
        "tasksRunning": 0,
        "tasksStaged": 0,
        "tasksUnhealthy": 0,
        "upgradeStrategy": {
            "maximumOverCapacity": 1,
            "minimumHealthCapacity": 1
        },
        "uris": [],
        "user": null,
        "version": "2016-06-22T19:09:41.482Z",
        "versionInfo": {
            "lastConfigChangeAt": "2016-06-22T19:05:46.147Z",
            "lastScalingAt": "2016-06-22T19:09:41.482Z"
        }
    }
]
