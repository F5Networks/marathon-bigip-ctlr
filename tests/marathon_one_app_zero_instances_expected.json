{"pools": [{"loadBalancingMode": "round-robin", "name": "server-app_80", "members": [], "monitors": ["/mesos/server-app_80_http"]}], "iapps": [], "l7Policies": [], "virtualServers": [{"profiles": [{"partition": "Common", "name": "http"}, {"partition": "Common", "name": "tcp"}], "destination": "/mesos/10.128.10.240:80", "sourceAddressTranslation": {"type": "automap"}, "name": "server-app_80", "connectionLimit": 0, "ipProtocol": "tcp", "enabled": true, "pool": "/mesos/server-app_80"}], "monitors": [{"portIndex": 0, "protocol": "HTTP", "name": "server-app_80_http", "timeoutSeconds": 20, "interval": 10, "intervalSeconds": 10, "ignoreHttp1xx": false, "gracePeriodSeconds": 5, "timeout": 21, "maxConsecutiveFailures": 1, "path": "/my_healthcheck", "type": "http"}]}
